SkillChat.Client/Utils/StackPanelMarginConverter.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing.Printing;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Avalonia;
+using Avalonia.Data.Converters;
+using ServiceStack;
+using SkillChat.Client.ViewModel;
+using SkillChat.Client.Views;
+
+namespace SkillChat.Client.Utils
+{
+    public class StackPanelMarginConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+           
+            switch ((MessageViewModel.StatusMessage)value)
+            {
+                case MessageViewModel.StatusMessage.MyFirstMessage:
+                    return new Thickness(20, 17, 0, 3);
+                case MessageViewModel.StatusMessage.MyNotFirstMessage:
+                    return new Thickness(20, 0, 0, 3);
+                case MessageViewModel.StatusMessage.SomeonesFirstMessage:
+                    return new Thickness(0, 17, 20, 3);
+                case MessageViewModel.StatusMessage.SomeonesNotFirstMessage:
+                    return new Thickness(0, 0, 20, 3);
+                default:
+                    return null;
+            }
+        }
```

Comment:
Можно написать элегантнее через switch expression
```suggestion
        public object Convert(object value, Type targetType, object parameter, CultureInfo culture) =>
            (MessageViewModel.StatusMessage)value switch
            {
                MessageViewModel.StatusMessage.MyFirstMessage => new Thickness(20, 17, 0, 3),
                MessageViewModel.StatusMessage.MyNotFirstMessage => new Thickness(20, 0, 0, 3),
                MessageViewModel.StatusMessage.SomeonesFirstMessage => new Thickness(0, 17, 20, 3),
                MessageViewModel.StatusMessage.SomeonesNotFirstMessage => new Thickness(0, 0, 20, 3),
                _ => null
            };
```

Reply:
Это круто, я не знал, что так можно